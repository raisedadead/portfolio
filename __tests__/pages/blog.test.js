
import React from 'react';
import { render } from 'react-testing-library';
import BlogMarkup from '../../src/pages/blog';

describe(`blog`, () => {
  it(`renders correctly`, () => {
    const {
      container: { firstChild }
    } = render(<BlogMarkup />);
    expect(firstChild).toMatchSnapshot();
  });
});
