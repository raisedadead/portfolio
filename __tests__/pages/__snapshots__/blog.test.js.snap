// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blog renders correctly 1`] = `
<div
  class="css-jqm3nq elghiqq0"
>
  <div
    class="css-11inlkn elghiqq0"
  >
    <nav
      class="css-1pdtlkx-PrimaryNav e1befqft0"
    >
      <a
        aria-label="Mrugesh Mohapatra's portfolio"
        class="e1befqft1 css-11ajdqu e70ypka2"
        data-index="0"
        href="/"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-1sp7cuy e70ypka0"
          >
            <span
              class="css-1me6504 e1i5p1da0"
            >
              home
            </span>
          </span>
        </span>
      </a>
      <a
        aria-label="Mrugesh Mohapatra's resumé"
        class="e1befqft1 css-11ajdqu e70ypka2"
        data-index="1"
        href="https://represent.io/mrugesh"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-1sp7cuy e70ypka0"
          >
            <span
              class="css-1me6504 e1i5p1da0"
            >
              resumé
            </span>
          </span>
        </span>
      </a>
      <a
        aria-label="Ask me anything"
        class="e1befqft1 css-11ajdqu e70ypka2"
        data-index="2"
        href="https://github.com/raisedadead/ama/"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-1sp7cuy e70ypka0"
          >
            <span
              class="css-1me6504 e1i5p1da0"
            >
              ask me anything
            </span>
          </span>
        </span>
      </a>
      <a
        aria-label="Blog"
        class="e1befqft1 css-11ajdqu e70ypka2"
        data-index="3"
        href="/blog"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-1sp7cuy e70ypka0"
          >
            <span
              class="css-1me6504 e1i5p1da0"
            >
              blog
            </span>
          </span>
        </span>
      </a>
    </nav>
  </div>
  <main>
    <main>
      <section
        class="css-11inlkn elghiqq0"
      >
        <div
          class="e7r6m60 css-wzh7ay elghiqq0"
        >
          <h4
            class="css-105a4qn e19le4zx0"
          >
            Did I not finsh creating this page? Yeah, I must have fallen asleep.
          </h4>
        </div>
      </section>
    </main>
  </main>
  <footer>
    <section
      class="css-11inlkn elghiqq0"
    >
      <nav
        class="css-1pdtlkx-PrimaryNav e1befqft0"
      >
        <a
          aria-label="Twitter"
          class="e1befqft1 css-ywx61d e70ypka2"
          data-index="0"
          href="https://twitter.com/raisedadead"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1me6504 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
        <a
          aria-label="GitHub"
          class="e1befqft1 css-ywx61d e70ypka2"
          data-index="1"
          href="https://github.com/raisedadead"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1me6504 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
        <a
          aria-label="LinkedIn"
          class="e1befqft1 css-ywx61d e70ypka2"
          data-index="2"
          href="https://linkedin.com/in/mrugeshm"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1me6504 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
      </nav>
      <h6
        class="css-1xj32bh e19le4zx0"
      >
         
        © 
        2019
         Mrugesh Mohapatra.
      </h6>
    </section>
  </footer>
</div>
`;
